torch/fx/proxy.py

```diff
@@ -259,6 +275,24 @@ def node(self):
     def __call__(self, *args, **kwargs):
         return self.tracer.create_proxy('call_method', self.attr, (self.root,) + args, kwargs)
 
+
+class FixedShapeParam(Proxy):
+    def __init__(self, node: Node, tracer, name, param):
+        super().__init__(node, tracer)
+        assert(isinstance(param, torch.nn.Parameter))
+        self.param = param
+        self.name = name
+
+    def __repr__(self) -> str:
+        return f'Parameter({self.name})'
```

Comment:
This should print the actual name of the class (i.e. s/Parameter/ParameterProxy/)